framework/src/st_things/things_stack/utils/things_network.c

```diff
@@ -304,6 +304,9 @@ void things_wifi_sta_connected(wifi_manager_result_e res)
 void things_wifi_sta_disconnected(void)
 {
 	THINGS_LOG_V(TAG, "T%d --> %s", getpid(), __FUNCTION__);
+#ifdef CONFIG_NETUTILS_NTPCLIENT
```

Comment:
We don't have to check CONFIG_DISABLE_SIGNALS??

Reply:
@Taejun-Kwon  
Good Point. 
I found some bugs in ntpclient about this MACRO.
i fixed it at PR #1930 
Thanks.

Reply:
@Taejun-Kwon, 
We aready define and use CONFIG_NETUTILS_NTPCLIENT in things sdk layer.
Can use CONFIG_DISABLE_SIGNALS, but user is able to feel any strange things.

Why are related functions binded each different definitions CONFIG_NETUTILS_NTPCLIENT and CONFIG_DISABLE_SIGNALS?

Can call sync_time_from_ntp when defined CONFIG_NETUTILS_NTPCLIENT?
Can call ntpc_stop when defined CONFIG_DISABLE_SIGNALS?

In my think,  we need to change things as use only one at least in same file.

After we had desided any way, I will reopen it.